File: VB/ViewModel/EFCore/ServerMode/EditIssueInfo.vb
Imports DevExpress.Mvvm
Imports System.Collections.Generic
Imports EFCoreIssues.Issues

Public Class EditIssueInfo
    Inherits BindableBase

    Public Sub New(ByVal context As IssuesContext, ByVal users As IList)
        Me.Context = context
        Me.Users = users
    End Sub

    Public ReadOnly Property Context As IssuesContext
    Public ReadOnly Property Users As IList
End Class

File: CS/ViewModel/EFCore/ServerMode/EditIssueInfo.cs
using DevExpress.Mvvm;
using DevExpress.Xpf.Grid;
using System.Collections;
using EFCoreIssues.Issues;

namespace EFCoreIssues {
    public class EditIssueInfo : BindableBase {
        public EditIssueInfo(IssuesContext dbContext, IList users) {
            DbContext = dbContext;
            Users = users;
        }
        public IssuesContext DbContext { get; }
        public IList Users { get; }
    }
}